sql/informix-sql/InformixSQLLexer.g4

```diff
@@ -257,5 +273,8 @@ SPACES: [ \u000B\t\r\n] -> channel(HIDDEN);
 
 UNEXPECTED_CHAR: .;
 
+fragment NEWLINE_EOF    : NEWLINE | EOF;
 fragment HEX_DIGIT: [0-9A-F];
 fragment DIGIT:     [0-9];
+fragment NEWLINE        : '\r'? '\n';
+fragment SPACE          : [ \t];
```

Comment:
I'm not sure why SPACE is defined. There are no applied occurrences of SPACE anywhere in either .g4, and SPACE cannot be used indirectly via value (as with string literals in a split grammar) because the parser cannot use lexer charsets. There are a slew of unused lexer literals in the grammar, e.g., COMMA and DIV.
```
$ bash /c/Users/Kenne/Documents/GitHub/g4-checks/unused-lexer.sh
Finding unused lexer symbols in grammars...
./desc.xml
InformixSQLLexer.g4:COMMA
InformixSQLLexer.g4:ASSIGN
InformixSQLLexer.g4:STAR
InformixSQLLexer.g4:PLUS
InformixSQLLexer.g4:MINUS
InformixSQLLexer.g4:TILDE
InformixSQLLexer.g4:PIPE2
InformixSQLLexer.g4:DIV
InformixSQLLexer.g4:MOD
InformixSQLLexer.g4:LT2
InformixSQLLexer.g4:GT2
InformixSQLLexer.g4:AMP
InformixSQLLexer.g4:PIPE
InformixSQLLexer.g4:LT
InformixSQLLexer.g4:LT_EQ
InformixSQLLexer.g4:GT
InformixSQLLexer.g4:GT_EQ
InformixSQLLexer.g4:EQ
InformixSQLLexer.g4:NOT_EQ1
InformixSQLLexer.g4:NOT_EQ2
InformixSQLLexer.g4:RETURNING
InformixSQLLexer.g4:TIES
InformixSQLLexer.g4:OTHERS
InformixSQLLexer.g4:DO
InformixSQLLexer.g4:NOTHING
InformixSQLLexer.g4:NUMERIC_LITERAL
InformixSQLLexer.g4:BIND_PARAMETER
InformixSQLLexer.g4:BLOB_LITERAL
InformixSQLLexer.g4:UNEXPECTED_CHAR
InformixSQLLexer.g4:NEWLINE_EOF
InformixSQLLexer.g4:SPACE
07/26-14:48:01 ~/g4-3633/sql/informix-sql
```